.markdownlint-cli2.cjs

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const sharedStandards = require(`${__dirname}/.markdownlint-cli2.shared.cjs`);
```

Comment:
Is the `__dirname` needed here? If memory is right, `require` automatically resolves relative paths based on `__dirname` ([docs](https://nodejs.org/docs/latest-v16.x/api/modules.html#requireid)).

Reply:
As discussed out of band, we're dropping `__dirname` for now. If issues arise, we can add it back and document why.